submissions/Zakkarat/html-forms/index.html

```diff
@@ -193,21 +193,20 @@ <h2 class="uploadText">Choose your languages pairings</h2>
                   <option value="Ukrainian" selected>Ukrainian</option>
                 </select>
               </div>
-              <img
-                tabindex="0"
+              <button><img
                 src="./img/waste-bin.svg"
                 class="deleteFile select-delete"
-              />
+              /></button>
               <button type="button" class="add-button">
                 Add
               </button>
             </div>
           </section>
           <section class="toneSelect">
             <div class="allTextDocumentUpload">
```

Comment:
Good practice to keeping the title of section and section content in one structured block.
What I mean: In your case, you have a block which contains titles and another one with content. Split them into two independent blocks in one row. Each block must contain title `h2` and content.
Feel free to ask me if you don't understand it.